tests/runtest.cmd

```diff
@@ -285,7 +298,64 @@ if errorlevel 1 (
    echo %__MsgPrefix%Test Run failed. Refer to the following:  
    echo     Html report: %__TestRunHtmlLog%  
 )  
+goto TestsDone
+
+REM =========================================================================================
+REM ===
+REM === Run CoreFX tests
+REM ===
+REM =========================================================================================
+:RunCoreFXTests
+set _CoreFXTestHost=%XunitTestBinBase%\testhost
+set _toolsDir=%__ProjectDir%\..\Tools
+set _dotnet=%_toolsDir%\dotnetcli\dotnet.exe
+
+set _RootCoreFXTestPath=%__TestWorkingDir%\CoreFX
+set _CoreFXTestUtilitiesOutputPath=%_RootCoreFXTestPath%\CoreFXTestUtilities
+set _CoreFXTestBinariesPath=%_RootCoreFXTestPath%\tests_downloaded
+set _CoreFXLogsDir=%__LogsDir%\CoreFX\%__BuildOS%.%__BuildArch%.%__BuildType%
+if not exist "%_CoreFXLogsDir%"  (mkdir "%_CoreFXLogsDir%")
+
+set _CoreFXTestSetupUtilityName=CoreFX.TestUtils.TestFileSetup
+set _CoreFXTestSetupUtility=%__ProjectFilesDir%\src\Common\CoreFX\TestFileSetup\%_CoreFXTestSetupUtilityName%.csproj
+
+call :ResolveDependencies
+if errorlevel 1 exit /b 1
+
+if defined __GenerateTestHostOnly (
+    exit /b 0
+)
+
+if not exist %_CoreFXTestHost%\dotnet.exe echo CoreFX test host not found, please run runtest.cmd again && exit /b 1
+
+set /p _CoreFXTestRemoteURL=< "%__ProjectFilesDir%\CoreFX\CoreFXTestListURL.txt"
+if not defined __CoreFXTestList ( set __CoreFXTestList=%__ProjectFilesDir%\CoreFX\TopN.CoreFX.Windows.issues.json )
```

Comment:
>TopN.CoreFX.Windows.issues.json [](start = 82, length = 31)

regarding "TopN.CoreFX.Windows.issues.json":
1. should there instead be a single file, with per-OS, per-arch, per-configuration (e.g., stress mode) exclusions (kind of like smarty's .lst file Categories), OR,
2. should this filename also have arch (e.g., x86, x64), and maybe config (e.g., stress mode) in it?

I would hope we could do #1, because if a test fails in all architectures/OS/stress combinations, I wouldn't want to add it to 15+ different files. Conversely, I hope we can disable on a relatively fine granularity.

Reply:
Both are completely reasonable - the helper projects are fairly simple and the schema is easily extensible. For the time being I've renamed the files to include architecture. 

If this proves to be a viable approach and the python script is fully deprecated, architectures/OS/stress combinations should definitely be included as well.